aioxmpp/muc/service.py

```diff
@@ -618,18 +633,28 @@ def _match_tracker(self, message):
         try:
             tracker = self._tracking_by_id[message.id_]
         except KeyError:
+            key = message.from_, _extract_one_pair(message.body)
```

Comment:
Dropping the from_ is problematic in the case of nick changes (and possibly reconnects).